### ISSUE TITLE
docset_links deleted from Kapeli

### ISSUE THREAD

**Issue by rawkode** [sunaku/dasht#23]
https://github.com/zealdocs/zeal/issues/294

**Comment by petRUShka** [comment]
Confirm. Right now it is impossible to install docsets:

```
$ dasht-docsets-install python                                                                                                                     
--2017-07-05 17:52:36--  https://kapeli.com/docset_links                                                                                                                                      
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'                                                                                                                                    
Resolving kapeli.com... 23.92.22.81                                                                                                                                                           
Connecting to kapeli.com|23.92.22.81|:443... connected.                                                                                                                                       
HTTP request sent, awaiting response... 301 Moved Permanently                                                                                                                                 
Location: https://kapeli.com/dash [following]                                                                                                                                                 
--2017-07-05 17:52:37--  https://kapeli.com/dash                                                                                                                                              
Reusing existing connection to kapeli.com:443.                                                                                                                                                
HTTP request sent, awaiting response... 304 Not Modified                                                                                                                                      
File '/home/user/.local/share/dasht/docsets/docset_links' not modified on server. Omitting download.

Installable docsets (0):
```

**Comment by alem0lars** [comment]
I'm having the same problem.

Any updates?

**Comment by sunaku** [comment]
Hey, thanks for reporting this issue. :+1: Unfortunately, my free time is scarce these days. :disappointed_relieved: PR is welcome.

**Comment by sunaku** [comment]
Fixed now in master at 4c650fbff7e311bd46b0e1cdbbf314aaee47ab9b.

**Comment by sunaku** [comment]
Released now in [version 2.2.0](https://github.com/sunaku/dasht/releases/tag/v2.2.0). :gift_heart:
